src/colp/components/manual_correction_report.py

```diff
@@ -0,0 +1,38 @@
+import streamlit as st
+import pandas as pd
+
+
+class ManualCorrection:
+    def __init__(self, data):
+        self.modifications_applied = data["modifications_applied"]
+        self.modifications_not_applied = data["modifications_not_applied"]
+
+    def __call__(self):
+        st.subheader("Manual Correction Report")
+
+        if (
+            self.modifications_applied is None
+            and self.modifications_not_applied is None
+        ):
+            st.info("No Manual Correction Table.")
+            return
+        self.display_manual_correction()
+
+    def display_manual_correction(self):
+        applied = self.modifications_applied
+        not_applied = self.modifications_not_applied
+        if not applied.empty:
+            count1 = applied.groupby("field").size()
+            count1.rename("Number of Records", inplace=True)
+            st.markdown("##### Numbers of Manual Corrections Applied By Field")
+            st.write(count1)
```

Comment:
The field column doesn't have a column label. I would assume you can fix this by adding an index name. If that doesn't work you may need to make the index it's own column and then give it a name

Reply:
Nice, I think this is much better. I think we can do even better and implement use `AgGrid` like the [outlier reports in pluto](https://github.com/NYCPlanning/data-engineering-qaqc/blob/7fe07adb1c8bec6eeff4dc941e900aad7741b621/src/pluto/pluto.py#L611). I think we discovered that module, meant to use it going forward and then kind of forgot about it 😬 
I opened an issue to use `AgGrid` for all tables that I can work on by my lonely self after you leave